base/src/CCNPPI.cpp

```diff
@@ -3,38 +3,856 @@
 #include "RawImagePlanarMetadata.h"
 #include "Frame.h"
 #include "Logger.h"
-#include "Utils.h"
 #include "AIPExceptions.h"
 #include "CCKernel.h"
-
 #include "npp.h"
 
 class CCNPPI::Detail
 {
 public:
-	Detail(CCNPPIProps &_props) : props(_props)
+	Detail(CCNPPIProps& _props) : props(_props)
 	{
 		nppStreamCtx.hStream = props.stream;
 	}
 
-	~Detail()
+	~Detail() {}
+
+	enum Imageformats
```

Comment:
put a comment - this enum has to be same as ImageMetadata enum

Reply:
made a comment